draft-quic-atsss-reqs.mkd

```diff
@@ -212,21 +212,24 @@ This document makes use of 3GPP specific terms defined in {{RFC6459}}, mainly th
 
 - UE (User Equipment): refers to the devices that are hosts with the
       ability to obtain Internet connectivity via a 3GPP network.
+     
+- Protocol Data Unit (PDU) Session: An association between the UE and the Data Network (DN) to carry the user data/traffic. 
```

Comment:
Thanks. Could you also define MA-PDU (not sure what is the distinction between PDU session and MA-PDU session)

Reply:
done